packages/node_modules/@ciscospark/plugin-wdm/src/device/device.js

```diff
@@ -250,6 +250,10 @@ const Device = SparkPlugin.extend({
 
       const timer = setTimeout(() => {
         this.spark.request({
+          headers: {
```

Comment:
It seems to run the test cases after removing this block. Hence removing these checks.